lib/polyphony/http/server/http1_adapter.rb

```diff
@@ -230,7 +230,7 @@ def format_headers(headers, body)
           headers.each do |k, v|
             next if k =~ /^:/
 
-            format_header_lines(k, v)
+            data << format_header_lines(k, v)
```

Comment:
This is the bug that prevented headers from being sent in the response.

Noticed it by testing the sample Cuba app, and comparing the behavior of `bin/poly` vs `rackup`.

When visiting `/`, the expected behavior is that the browser follows the redirection to `/hello`. Since the `Location` header was not being rendered in the response, when serving with `bin/poly` the browser just stayed there 😄

Reply:
This fix should be put in a separate commit.

Reply:
Makes sense, I can create a separate PR if that works.